documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,188 @@
+
+### Authentication Request
+
+This CAMARA document makes the login_hint parameter REQUIRED if CIBA is used in a CAMARA context.
+
+This CAMARA document clarifies the values used in login_hint in the following way:
+
+    * **_tel_**
+
+      For phone numbers. The `login_hint` must be a tel URI as defined in [RFC 3966](https://www.rfc-editor.org/info/rfc3966) for global phone numbers without visual separators in [E.164](https://www.itu.int/rec/T-REC-E.164-201011-I/en) format. For example, `tel:+34666666666`.
+
+    * **_ipport_**
+
+      For IPv4 and IPv6 addresses, that can optionally include a port. For example, `ipport:80.90.34.2:16790`, `ipport:80.90.34.2`, `ipport:[2001:db8::1]:8080` or `ipport:[2001:db8::1]`.
+
+
+The client MUST authenticate with the authorization server as described in [Client Authentication Section](#client-authentication).
+
+## Client Credentials Flow
+
+The [OAuth 2.0 Client Credentials](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4) grant type is used to obtain a 2-legged Access Token that does not represent a user. This grant type can only be used when no personal user data is processed, and it is only a valid option to access the CAMARA APIs for these specific scenarios.
+
+## Handling of acr_values
+
+OIDC specifies in [Mandatory to Implement Features for All OpenID Providers](https://openid.net/specs/openid-connect-core-1_0.html#ServerMTI) that OpenId Providers MUST implement support for acr_values.
+
+> Authentication Context Class Reference
+>>    OPs MUST support requests for specific Authentication Context Class Reference values via the acr_values parameter, as defined in Section 3.1.2. (Note that the minimum level of support required for this parameter is simply to have its use not result in an error.)
+
+OIDC also defines that the parameter acr_values is OPTIONAL and does not specify possible values. This leads to interoperability issues.
+
+This documents defines that Camara OpenId Providers MUST ignore the parameter acr_values. 
+
+This document defines that Camara Clients SHOULD not use the acr_values parameter. 
+
+> To foster interoperability a future version of this document might define values for the acr_values parameter acceptable in Camara.
+
+## Access Token Request
+
+This CAMARA document makes the scope parameter REQUIRED for the OAuth2 Client Credentials Grant.
+
+The client MUST authenticate with the authorization server as described in [Client Authentication Section](#client-authentication).
+
+## The Scope Parameter
+
+
+Scope values determine the specific CAMARA services being requested by the Service Provider, subject to the SP being registered to use those services. The scope values must be documented in the API OAS files by all Camara API subprojects. This document does not change OIDC definitions of scope values.
+
```

Comment:
The access token is conveyed to the RS, because otherwise the RS cannot decide whether access should be granted or not. These authorization enforcement must also include scopes if the API defines scopes.

If after an RS has made the authorization decision scopes and other information from the access token is not forwarded to the actual API implementation, then this is not a Camara problem to be handled in ICM, I think.